client/dev.sh

```
@@ -0,0 +1,6 @@
+#!/bin/bash
+. ~/.nvm/nvm.sh  # Load nvm
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)